fem/convergence.cpp

```diff
@@ -29,7 +29,7 @@ double ConvergenceStudy::GetNorm(GridFunction *gf, Coefficient *scalar_u,
 {
    bool norm_set = false;
    double norm=0.0;
-   int order = gf->FESpace()->GetOrder(0);
+   int order = gf->FESpace()->GetElementDegree(0);
```

Comment:
```suggestion
   int order = gf->FESpace()->GetMaxElementOrder();
```
Would this be better?

Reply:
This is a good idea. I'm just not sure I want to risk breaking something since "order" and "degree" may be different.

Reply:
It looks like the order is just used to determine the quadrature rule, so this should be OK, right?

Reply:
OK, this case is probably safe. Done.